packages/node_modules/nav-frontend-skjema/sample/_bekreftcheckbokspanel.sample.js

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line
```

Comment:
Fint eksempel, men tenker kanskje vi kanskje får enda mer utviklerdok utifra å ikke bruke `no-transpilation.js`-delen av APIet her?

Forslag:
```
export default generateSample({
    baseType: BekreftCheckboksPanelExample,
    children: (<span>Her kan du sende inn html. For eksempel en <a href="/lenke">lenke</a></span>),
    attrs: {
        label: 'Jeg godtar',
        onChange: () => {}
    }
})
```

Foreslår også å droppe å bruke `<p>` og heller bare ta en `<span>` for å slippe den litt uheldige margin-en over/under.

Reply:
`BekreftCheckboksPanelExample` er en dum-komponent, så om vi skal vise frem funksjonaliteten så må vi nok bruke `no-transpilation`-delen. Evt. må vi eksponerer `checked` som en prop til appen, slik at man kan huke av en checkbox for å toggle visningen.

Reply:
Har oppdatert koden med en span i eksempelet. La også til en liten margin under innholdet.